support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/ApproverAction/summary.jelly

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License
+
+Copyright (c) 2013-2014, CloudBees, Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+ <j:jelly xmlns:j="jelly:core" xmlns:t="/lib/hudson">
+   <t:summary icon="orange-square.png">
+        ${%approved_by_user(it.userId, it.userName, rootURL)}.
```

Comment:
I understand your point - but then, I am not sure why the code above works, [as well as] (https://github.com/jenkinsci/jenkins/blob/419d670cd0f672b0b3eba6ad4bcc97e8c6948424/core/src/main/resources/hudson/model/Cause/UserIdCause/description.jelly). 
![screen shot 2015-03-04 at 12 24 34](https://cloud.githubusercontent.com/assets/5398781/6482732/79a4e0c6-c269-11e4-9399-b10880b8f801.png)